fe/fe-core/src/main/java/org/apache/doris/cloud/catalog/CloudEnv.java

```diff
@@ -175,14 +212,19 @@ protected void getClusterIdAndRole() throws IOException {
             try {
                 nodeInfoPB = getLocalTypeFromMetaService();
             } catch (Exception e) {
-                LOG.warn("failed to get local fe's type, sleep 5 s, try again. exception: {}", e.getMessage());
+                LOG.warn("failed to get local fe's type, sleep {} s, try again. exception: {}",
+                        e.getMessage(), Config.resource_not_ready_sleep_seconds);
```

Comment:
wrong position of the 2 arguments

```suggestion
                        Config.resource_not_ready_sleep_seconds, e.getMessage());
```